From 011b3515c97ace099cd9e224ba2257aeb87b7715 Mon Sep 17 00:00:00 2001
Message-Id: <011b3515c97ace099cd9e224ba2257aeb87b7715.1426116068.git.feitong.yi@intel.com>
In-Reply-To: <581bcb8c8802c2b763f5abc462a37e22e0195720.1426116068.git.feitong.yi@intel.com>
References: <581bcb8c8802c2b763f5abc462a37e22e0195720.1426116068.git.feitong.yi@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Mon, 23 Feb 2015 19:51:16 +0000
Subject: [PATCH 4/5] SQUASHME! [VPG]: include/linux: Remove duplicate code

Remove code duplication which is stopping i915 compiling as a module with
the older compilers used when doing a static analysis tool scan.

Fumble was introduce in:
commit 4b5e69133e5bd696823207f9ac90b206dfae6b8c
Author: Fei Yang <fei.yang@intel.com>
Date:   Fri Jun 13 14:31:28 2014 -0700

    EXTCON: add drivers to suport Dc/Crc/Smsc3750

Issue: GMINL-6529
Change-Id: Ib7772c61f728eb186a2a271259965b62009dbc6d
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 include/linux/extcon.h |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/include/linux/extcon.h b/include/linux/extcon.h
index f618658..3e4d4f8 100644
--- a/include/linux/extcon.h
+++ b/include/linux/extcon.h
@@ -306,11 +306,6 @@ static int extcon_find_cable_type(struct extcon_dev *edev, int index)
 	return 0;
 }
 
-static int extcon_find_cable_type(struct extcon_dev *edev, int index)
-{
-	return 0;
-}
-
 static inline int extcon_get_cable_state_(struct extcon_dev *edev,
 					  int cable_index)
 {
-- 
1.7.9.5

